.circleci/src/jobs/deliver_administration.yml

```diff
@@ -0,0 +1,44 @@
+docker:
+  - image: cimg/base:2022.09
+parameters:
+  production:
+    description: Whether builds are delivered to production or beta.
+    type: boolean
+steps:
+  - run:
+      name: Install Curl
+      command: |
+        sudo apt update
+        sudo apt install curl -y
+  - attach_workspace:
+      at: /tmp/workspace
+  - add_ssh_keys:
+      fingerprints:
+        - "a1:3f:a7:c3:ff:12:40:1d:85:de:a7:ab:12:3f:cc:05"
+  - when:
+      condition: << parameters.production >>
+      steps:
+        - run:
+            name: SFTP upload package
+            command: |
+              echo "|1|iikuvSrIo3wkj+EqUgLRMsAq6yk=|r9bSjkawWFa94b45qE/se5Oio5k= ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIED3MwobbCs+ENMLXdyqlJb3bJ26TuIYt977TA3NrN66" >> known_hosts
+              echo "Uploading: " /tmp/workspace/debs/administration/*.deb
+# will be enabled with #1271
+#              sftp -b - -o UserKnownHostsFile=known_hosts ci@entitlementcard.tuerantuer.org:/local-apt-repository/ \<<< "put -r /tmp/workspace/debs/administration/*.deb"
+#        - run:
+#            name: Install package via webhook
+#            command: |
+#              https://webhook.entitlementcard.tuerantuer.org/hooks/install-admin-$EAK_WEBHOOK
+  - unless:
+      condition: << parameters.production >>
+      steps:
+        - run:
+            name: SFTP upload package
+            command: |
+              echo "|1|dkYQrdGB1QML0o+POL3QzAkBbek=|b4Tm0Ymh82UKyZPJfVKy4t+MFV8= ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIONuARu18Fktz+j4QosI+nRqgMnFOMgE7OZLuTOwgZ0k" >> known_hosts
+              echo "Uploading: " /tmp/workspace/debs/administration/*.deb
+              sftp -b - -o UserKnownHostsFile=known_hosts ci@entitlementcard-test.tuerantuer.org:/local-apt-repository/ \<<< "put -r /tmp/workspace/debs/administration/*.deb"
```

Comment:
If we write to the correct / default known host file, which is `~/. ssh/known_hosts` we can get rid of manually specifying the `UserKnownHostsFile`

In integreat, we just disable the host key checking. Why have you decided to add the checks here?

Reply:
Hm i don't see the big advantage to write to the servers `known_hosts` tbh. This way is flexible and ensures that its temporary. I think i figured this out with @svenseeberg 
disable host key checking seems to be a red flag, because you can do host spoofing but i'm not an expert. Just read some blogs.